ClinPred.pm

```diff
@@ -87,11 +92,16 @@ sub run{
   my $vf = $tva->variation_feature;
   my $allele = $tva->variation_feature_seq;
   return {} unless $allele =~ /^[ACGT]$/;
+  
+
+  my ($vf_start, $vf_end) = ($vf->{start}, $vf->{end});
+  ($vf_start, $vf_end) = ($vf->{end}, $vf->{start}) if ($vf->{start} > $vf->{end});
```

Comment:
It would be easier to read if you only use the variables $vf_start and $vf_end to check for an insertion:
($vf_start, $vf_end) = ($vf_end, $vf_start) if ($vf_start > $vf_end);